src/service/viewport-impl.js

```diff
@@ -750,6 +750,10 @@ export class ViewportBindingNatural_ {
     if (this.win.document.defaultView) {
       waitForBody(this.win.document, () => {
         this.win.document.body.style.overflow = 'visible';
+        setStyles(this.win.document.body, {
+          overflowX: 'hidden',
+          overflowY: 'auto',
+        });
```

Comment:
CSS or otherwise are ok. But two things we need to do:

1. Put this under experiment
2. Only do any customization when `#webview=1` is specified.
3. We definitely, under no circumstances, can have `overflow-y: auto` on body in this mode. If we can't get around without it, then we need to look for another way, such as instead switching to `ViewportBindingNaturalIosEmbed_` for webviews.